src/project/project.go

```diff
@@ -154,3 +159,15 @@ func detectName(path string) (string, error) {
 
 	return name, nil
 }
+
+func GetAppDir(projectCtx *ProjectCtx) string {
+	return filepath.Join("/usr/share/tarantool", projectCtx.Name)
```

Comment:
nit: seems it's a bit inconsistent `<path>/` and `<path>`.

I suggest you to add slash at the end of `/usr/share/tarantool` or drop for paths below.